Gemfile.lock

```diff
@@ -109,14 +111,14 @@ GEM
     pathutil (0.16.2)
       forwardable-extended (~> 2.6)
     public_suffix (4.0.6)
-    rack (2.1.4)
+    rack (1.6.13)
```

Comment:
Codacy found an issue: [Directory traversal in Rack::Directory app bundled with Rack (CVE-2020-8161)](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6185603)